config/config.go

```diff
@@ -162,10 +155,10 @@ func NewStoresConfig() *StoresConfig {
 }
 
 // Load loads a sops config file into a temporary struct
-func (f *configFile) load(bytes []byte) error {
+func (f *ConfigFile) load(bytes []byte) error {
 	err := yaml.Unmarshal(bytes, f)
 	if err != nil {
-		return fmt.Errorf("Could not unmarshal config file: %s", err)
+		return fmt.Errorf("could not unmarshal config file: %s", err)
```

Comment:
This was also flagged by `go-staticcheck`.

Reply:
This is a change that should go into a feature release and not into a bugfix release since it changes the observable behavior a bit and isn't a bugfix. Because of that I would keep it separate from the other changes.